From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Logan Bussell <loganbussell@microsoft.com>
Date: Thu, 12 May 2022 14:32:59 -0700
Subject: [PATCH] Disable warning CS0618

This is needed because IOperation.Children is deprecated in
Microsoft.CodeAnalysis 4.3.0, but aspnetcore references version 4.2.0.

Requires https://github.com/dotnet/source-build/issues/2482 in order to eliminate this patch
---
 .../src/Microsoft.AspNetCore.Mvc.Api.Analyzers.csproj            | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/Mvc/Mvc.Api.Analyzers/src/Microsoft.AspNetCore.Mvc.Api.Analyzers.csproj b/src/Mvc/Mvc.Api.Analyzers/src/Microsoft.AspNetCore.Mvc.Api.Analyzers.csproj
index 0b94ad2a8f..1b5cb9d907 100644
--- a/src/Mvc/Mvc.Api.Analyzers/src/Microsoft.AspNetCore.Mvc.Api.Analyzers.csproj
+++ b/src/Mvc/Mvc.Api.Analyzers/src/Microsoft.AspNetCore.Mvc.Api.Analyzers.csproj
@@ -9,6 +9,7 @@
     <IncludeBuildOutput>false</IncludeBuildOutput>
     <GenerateDocumentationFile>false</GenerateDocumentationFile>
     <NuspecFile>$(MSBuildProjectName).nuspec</NuspecFile>
+    <NoWarn>$(NoWarn);CS0618</NoWarn>
   </PropertyGroup>
 
   <ItemGroup>
